reports/src/main/java/io/jmix/reports/ReportSecurityManager.java

```diff
@@ -65,8 +65,8 @@ public void applySecurityPolicies(LoadContext lc, @Nullable String screen, @Null
                 lc.getQuery().setParameter(paramName, wrapCodeParameterForSearch(role.getCode()));
                 transformer.addWhereAsIs(roleCondition.toString());
```

Comment:
Pull the method out of the loop to generate jpql correctly. 
`transformer.addWhereAsIs(roleCondition.toString());`